third-party/cmake/LibvtermCMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(libvterm LANGUAGES C)
```

Comment:
> But I can merge everything into a single file.

Now that I look at it, not sure what is the correct way to create a new file with a specific contents from within the recipe file, e.g. file(write ...) is not very ergonomic for large files.